version/version.go

```diff
@@ -1,6 +1,8 @@
 package version
 
 var (
-	Version   = "v2"
+	// Version defines the current version of keda
+	Version = "v2"
```

Comment:
`sed` command in the Makefile needs to be probably updated because of this change:
https://github.com/kedacore/keda/blob/598f1a4985fe4965a579695f195701472ae079ea/Makefile#L81

Reply:
Thanks for the feedback! I tested this as it is and it works as it should. I think thats because the pattern used matches the full assignment statement, not just the variable name.

To test I used the below in the Makefile

```
.PHONY: get-version
get-version:
	
	@sed -i 's@Version   =.*@Version   = "$(VERSION)"@g' ./version/version.go;
	@echo $(VERSION);
```